packages/help-text/src/HelpTextManager.ts

```diff
@@ -43,7 +43,10 @@ export class HelpTextManager {
         // `pass-through-help-text-${this.instanceCount}` makes the slot effectively unreachable from
         // the outside allowing the `help-text` slot to be preferred while `negative === false`.
         return html`
-            <div id=${ifDefined(this.isInternal ? this.id : undefined)}>
+            <div
+                id=${ifDefined(this.isInternal ? this.id : undefined)}
```

Comment:
Does a truthy `this.isInternal` always specify that an `id` will be present?

Reply:
Yes it does. And, while it's _technically_ possible that a custom element consumer would _unset_ the value of ID, but that would be highly irregular. Without negative intent, the `id` as set in the constructor (note that this is a helper class and not a custom element class) would always be available for use here.

As this isn't new functionality, if there's a concern we want to address here, it would be better to track it in an issue for a later update.